src/vs/workbench/browser/parts/panel/media/panelpart.css

```diff
@@ -110,7 +110,12 @@
 	text-align: center;
 	display: inline-block;
 	box-sizing: border-box;
-	}
+}
+
+/* Dropdown */
+.monaco-workbench .panel .monaco-action-bar .select-container .monaco-select-box {
```

Comment:
Shouldn't this move into the rule we already have for `.monaco-select-box` below?

Reply:
This was specific to the panel, but I just double-checked and this rule doesn't need to be here anymore so I'll remove it.